src/lwc/signals/core.ts

```diff
@@ -463,4 +534,8 @@ function $resource<ReturnType, Params>(
   };
 }
 
-export { $signal, $effect, $computed, $resource };
+function isASignal(anything: unknown): anything is Signal<unknown> {
+  return !!anything && (anything as Signal<unknown>).brand === Symbol.for("lwc-signals");
```

Comment:
Since you are using the Symbol in 2 places now, probably should become a constant